src/extension/background.re

```diff
@@ -72,6 +72,35 @@ Chrome.ContextMenus.create {
   "onclick": fun _ tab => toggleConversion tab##id
 };
 
+let convertExt ext => switch ext {
+  | "ml"  => "mli"
+  | "mli" => "ml"
+  | "re"  => "rei"
+  | "rei" => "re"
+  | ext => ext
+};
+let getNewURL url => 
+  switch (Js.Array.pop @@ Js.String.split "." url) {
+  | None => url
+  | Some ext => Js.String.replace ("." ^ ext) ("." ^ (convertExt ext)) url
+  };
+
+Chrome.Commands.addListener (
+  fun command => {
+    Js.log command;
+    if(command == "toggle_between_interface_and_implementation") {
+      Chrome.Tabs.query {"active": Js.true_, "currentWindow": Js.true_} {fun activeTab => {
+      let currentTab = activeTab.(0);
```

Comment:
`activeTab`, a singular, isn't a very good name for an array. The chrome API docs call it `result`. You could also just destructure it directly: `fun [currentTab] => ...`